cfgov/scripts/update_categories.py

```diff
@@ -0,0 +1,52 @@
+from v1.models.base import CFGOVPage
+from v1.models.related_posts_category import RelatedPostsCategory
+
+
+def run():
+    for page in CFGOVPage.objects.all():
+        sidefoot = page.sidefoot
+        if sidefoot:
+            stream_data = sidefoot.stream_data
+            related_posts = filter(
+                lambda item: item['type'] == 'related_posts',
+                stream_data
+            )
+            if related_posts:
+                cats = related_posts[0]['value']['specific_categories']
+                for cat in cats:
+                    if not cat:
+                        continue
+                    if cat == 'Policy &amp; Compliance':
+                        new_cat = RelatedPostsCategory(
+                            page=page,
+                            name='policy-compliance'
+                        )
+                        new_cat.save()
+                    elif cat == 'Info for Consumers':
+                        new_cat = RelatedPostsCategory(
+                            page=page,
+                            name='info-for-consumers'
+                        )
+                        new_cat.save()
+                    elif cat == 'Data, Research &amp; Reports':
+                        new_cat = RelatedPostsCategory(
+                            page=page,
+                            name='data-research-reports'
+                        )
+                        new_cat.save()
+                    elif cat == 'At the CFPB':
+                        new_cat = RelatedPostsCategory(
+                            page=page,
+                            name='at-the-cfpb'
+                        )
+                        new_cat.save()
+                    elif cat == 'Press Release':
+                        new_cat = RelatedPostsCategory(
+                            page=page,
+                            name='press-release'
```

Comment:
Last I checked there were no op-ed, speech, or testimony categories in prod data, but I'll add them in here anyway to be comprehensive.